Projekty/Projekt1/Jung_Kaczmarek_Marciniak/Milestone_2.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>To aż się prosi o wykres (najlepiej z zaznaczonymi odchyleniami standardowymi). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>